package.json

```diff
@@ -51,6 +51,7 @@
     "cozy-scripts": "5.13.0",
     "cozy-ui": "^58.2.0",
     "date-fns": "^1.30.1",
+    "humanize-duration": "3.27.1",
```

Comment:
what is the size added by this lib to the bundle?

Reply:
![image](https://user-images.githubusercontent.com/67680939/145990288-c7cbd4e6-fdbc-4037-ad30-25c7b2cc2990.png)

Reply:
6ko just to display a week ago :'( this is insane. We should never tolerate this...